lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,56 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        cube.add_dim_coord(
+            DimCoord(standard_name='time',
+                     units=Unit('days since 0001-01-01 00:00:00',
+                                calendar='gregorian'),
+                     points=np.arange(2),),
+            0)
+        cube.add_dim_coord(DimCoord(standard_name='longitude',
+                                    units='degrees',
+                                    points=np.arange(3),), 1)
```

Comment:
I just don't think there is a need to change it, I think @cpelley just meant to change from using points as a keyword to using it as a positional argument, otherwise this is fine.